src/Sylius/Bundle/ShopBundle/Resources/views/Account/Order/Grid/Action/pay.html.twig

```diff
@@ -1,7 +1,7 @@
 {% import '@SyliusUi/Macro/buttons.html.twig' as buttons %}
 
-{% set path = options.link.url|default(path(options.link.route|default(grid.requestConfiguration.getRouteName('show')), options.link.parameters|default({'id': data.id}))) %}
-
-{% if data.paymentState in ['awaiting_payment'] %}
-    {{ buttons.default(path, action.label, null, action.options.icon|default('credit card alternative'), 'blue') }}
-{% endif %}
+{% set path = options.link.url|default(path(
+    options.link.route|default(grid.requestConfiguration.getRouteName('show')),
+    options.link.parameters|default({'id': data.id})
+)) %}
+{{ buttons.default(path, action.label, null, action.options.icon|default('credit card alternative'), 'blue') }}
```

Comment:
The behat tests expect that every order has a payment button in the grid and then when the customer clicks this button, they are told they can't do that. Maybe we should fix the test to only expect a button when the order is still modifiable. (Eg. When the order is canceled don't try to click on the change payment button)

Reply:
Reverted.